homeassistant/components/xiaomi_miio/const.py

```diff
@@ -0,0 +1,48 @@
+"""Constants for the Xiaomi Miio component."""
+DOMAIN = "xiaomi_miio"
+
+# Fan Services
+SERVICE_SET_BUZZER_ON = "set_buzzer_on"
+SERVICE_SET_BUZZER_OFF = "set_buzzer_off"
+SERVICE_SET_LED_ON = "set_led_on"
+SERVICE_SET_LED_OFF = "set_led_off"
+SERVICE_SET_CHILD_LOCK_ON = "set_child_lock_on"
+SERVICE_SET_CHILD_LOCK_OFF = "set_child_lock_off"
+SERVICE_SET_LED_BRIGHTNESS = "set_led_brightness"
+SERVICE_SET_FAVORITE_LEVEL = "set_favorite_level"
+SERVICE_SET_AUTO_DETECT_ON = "set_auto_detect_on"
+SERVICE_SET_AUTO_DETECT_OFF = "set_auto_detect_off"
+SERVICE_SET_LEARN_MODE_ON = "set_learn_mode_on"
+SERVICE_SET_LEARN_MODE_OFF = "set_learn_mode_off"
+SERVICE_SET_VOLUME = "set_volume"
+SERVICE_RESET_FILTER = "reset_filter"
+SERVICE_SET_EXTRA_FEATURES = "set_extra_features"
+SERVICE_SET_TARGET_HUMIDITY = "set_target_humidity"
+SERVICE_SET_DRY_ON = "set_dry_on"
+SERVICE_SET_DRY_OFF = "set_dry_off"
+
+# Light Services
+SERVICE_SET_SCENE = "set_scene"
+SERVICE_SET_DELAYED_TURN_OFF = "set_delayed_turn_off"
+SERVICE_REMINDER_ON = "reminder_on"
```

Comment:
I don't think that we can just strip the platform prefix here. Because of the many different platforms, it's confusing to know which service belongs to what. I wonder if we need to prefix them with `light_` etc

Reply:
I can make that update - originally they were the integration name, which would have been redundant.

Reply:
I made the update and thought I pushed it but I may have just committed it locally. I will have to push the service name changes on Sunday

Reply:
This was resolved